docs/en/reference/additionally/actions.mdx

```diff
@@ -173,90 +159,82 @@ You can get the following parameters:
 
 ---
 
-## actions.getDays()
+## onCreateDateEls()
 
 `Type: Function`
 
 `Default: null`
 
-`Options: getDays(day, date, HTMLElement, HTMLButtonElement, self) => void | null`
+`Options: onCreateDateEls(day, date, mainElement, HTMLButtonElement, self) => void | null`
 
 ```js
 new VanillaCalendar('#calendar', {
-  actions: {
-    getDays(day, date, HTMLElement, HTMLButtonElement, self) {},
-  },
+  onCreateDateEls(day, date, mainElement, HTMLButtonElement, self) {},
 });
 ```
 
 This method triggers during the calendar initialization and any changes. It provides access to information about each day and its HTML element. You can get the following parameters:
 - `day` - the day number;
 - `date` - the full date in the `YYYY-MM-DD` format;
-- `HTMLElement` - the parent (wrapper) HTML element of the day;
+- `mainElement` - the parent (wrapper) HTML element of the day;
 - `HTMLButtonElement` - the child (button) HTML element of the day.
 - `self` - reference to the initialized calendar.
 
 ---
 
-## actions.getMonths()
+## onCreateMonthEls()
 
 `Type: Function`
 
 `Default: null`
 
-`Options: getMonths(month: number, HTMLElement: HTMLElement, self: IVanillaCalendar) => void | null`
+`Options: onCreateMonthEls(month: number, mainElement: HTMLElement, self: IVanillaCalendar) => void | null`
 
 ```js
 new VanillaCalendar('#calendar', {
-  actions: {
-    getMonths(month, HTMLElement, self) {},
-  },
+  onCreateMonthEls(month, mainElement, self) {},
 });
 ```
 
 This method is triggered when the calendar type is set to `'month'`. The calendar type also becomes `'month'` when the user clicks on the month header or during initialization with the `type = 'month'` parameter. It provides access to information about each month and its HTML element. You can retrieve the following parameters:
 - `month` - month number (starts at 0);
-- `HTMLElement` - HTML element of the month;
+- `mainElement` - HTML element of the month;
 - `self` - reference to the initialized calendar.
 
 ---
 
-## actions.getYears()
+## onCreateYearEls()
 
 `Type: Function`
 
 `Default: null`
 
-`Options: getYears(year: number, HTMLElement: HTMLElement, self: IVanillaCalendar) => void | null`
+`Options: onCreateYearEls(year: number, mainElement: HTMLElement, self: IVanillaCalendar) => void | null`
```

Comment:
I missed a type here where `IVanillaCalendar` should be using the new `Calendar`